app/services/authn_context_resolver.rb

```diff
@@ -98,6 +100,13 @@ def acr_result_without_sp_defaults
     @acr_result_without_sp_defaults ||= Vot::Parser.new(acr_values: acr_values).parse
   end
 
+  def maybe_force_ial2_acr_result(result)
+    return result if !service_provider&.force_ial2_allowed?
+    return result if !result.identity_proofing?
+
+    result.with(facial_match?: true)
```

Comment:
Here I think you will want all of the IAL2 requirements:

https://github.com/18F/identity-idp/blob/eacb20d22499e6276df81d73a8ab22b2e5907119/app/services/vot/acr_component_values.rb#L34-L35

I think you'll already have `aal2` from the `identity_proofing` component, but it probably doesn't hurt to be explicit. I think you'll definitely need `two_pieces_of_fair_evidence`.